bootstrap/filters.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// Override the title
+add_filter('the_title', function($title, $post_id) {
+	if( $title_override = get_field('banner_title_override', $post_id) ) {
+        $title = trim($title_override) ? trim($title_override) : get_the_title($post_id);
```

Comment:
simplify  and use the $title param to check against

Reply:
Good suggestions. On it...